httplib2shim/__init__.py

```diff
@@ -32,8 +32,10 @@
 
 def _default_make_pool(http, proxy_info):
     """Creates a urllib3.PoolManager object that has SSL verification enabled
-    and uses the certifi certificates."""
+    and uses the certifi certificates.
 
+    Args:
+        proxy_info: A function or None."""
```

Comment:
since it's possible for proxy_info to be none, you might as well make it an optional arg `_default_make_pool(http, proxy_info=None` and update this docstring to say `proxy_info (Callable): An optional callable that returns proxy info.`